.travis.yml

```diff
@@ -9,24 +9,27 @@ script:
   - ruby script/jack_hammer -t 2000
 matrix:
   include:
-    - rvm: 2.1.8
+    - rvm: 2.2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-3.2.x
       env: DB=mysql
-    - rvm: 2.1.8
+    - rvm: 2.1.9
       gemfile: spec/support/gemfiles/Gemfile.rails-4.2.x
       env: DB=mysql
-    - rvm: 2.2.4
-      gemfile: spec/support/gemfiles/Gemfile.rails-4.2.x
+    - rvm: 2.2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.0.x
       env: DB=mysql
-    - rvm: 2.3.0
+    - rvm: 2.3.1
       gemfile: spec/support/gemfiles/Gemfile.rails-4.1.x
```

Comment:
With the release of Rails 5.0 [Rails 4.1](http://guides.rubyonrails.org/maintenance_policy.html) and below will essentially be unsupported. Does it still make sense to spec against them?